src/wh_server.c

```diff
@@ -84,6 +85,7 @@ int wh_Server_Cleanup(whServerContext* server)
 
     (void)wh_CommServer_Cleanup(server->comm);
     (void)wh_Nvm_Cleanup(server->nvm);
+    (void)wc_FreeRng(server->crypto->rng);
```

Comment:
All of the crypto component init and shutdown should be self contained.  No changes necessary.